cooper_client_react.md

```diff
@@ -0,0 +1,383 @@
+# Cooper client in React
+
+First we need to create the application. Run:
+
+`npx create-react-app cooper_client`
+
+We are going to setup the acceptance testing first. Like we did in the BMI Calculation project, we are going to use `jest-puppeteer`.
+
+We need to configure it now. Run:
+```shell
+$ touch jest-puppeteer.config.js
+$ touch jest.config.js
+```
+And add this to them:
+
+```js
+// jest.config.js
+module.exports = {
+    verbose: true,
+    preset: "jest-puppeteer",
+    testRegex: ".feature\\.js$"
+}; 
+```
+
+```js
+module.exports = {
+  launch: {
+      headless: false,
```

Comment:
In the bmi guide you use `headless: process.env.CI === 'true',` any reason why you dont use it here?

Reply:
We have been doing so many changes to this file over the time so I cant really answer the exact reason why we dont use it here. But I will update this file with the latest solution that seems to work well.